packages/gcc10.rb

```diff
@@ -69,128 +76,338 @@ def self.build
+      Dir.chdir "#{CREW_DEST_MAN_PREFIX}/man1" do
+        Dir.glob("*-#{gcc_version}.1*").each do |f|
+          @basefile = f.gsub("-#{gcc_version}", '')
+          FileUtils.ln_sf f, @basefile
+        end
+      end
+
+      # Only make links to unversioned in postinstall
+      Dir.chdir "#{CREW_DEST_PREFIX}/bin/" do
+        Dir.glob("#{gcc_arch}-*-#{gcc_version}").each do |f|
+          # @basefile_nover=f.split(/-#{gcc_version}/, 2).first
+          # puts "Symlinking #{f} to #{@basefile_nover}"
+          # FileUtils.ln_sf f, @basefile_nover
+          @basefile_noarch = f.split(/#{gcc_arch}-/, 2).last
+          puts "Symlinking #{f} to #{@basefile_noarch}"
+          FileUtils.ln_sf f, @basefile_noarch
+        end
+        # Dir.glob("*-#{gcc_version}").each do |f|
+        # @basefile_nover=f.split(/-#{gcc_version}/, 2).first
+        # FileUtils.ln_sf f, @basefile_nover
+        # end
+        # # many packages expect this symlink
+        # FileUtils.ln_sf "gcc-#{gcc_version}", 'cc'
+      end
+      # libgomp.so conflicts with llvm
+      @deletefiles = %W[#{CREW_DEST_LIB_PREFIX}/libgomp.so]
+      @deletefiles.each do |f|
+        FileUtils.rm f if File.exist?(f)
+      end
+    end
+  end
+
+  def self.postinstall
+    # Only make links to unversioned in postinstall
+    gcc_arch = `gcc-#{version} -dumpmachine`.chomp
+    gcc_version = version.split('-')[0]
+    gcc_dir = "gcc/#{gcc_arch}/#{gcc_version}"
+    Dir.chdir "#{CREW_PREFIX}/bin/" do
+      Dir.glob("#{gcc_arch}-*-#{gcc_version}").each do |f|
+        @basefile_nover = f.split(/-#{gcc_version}/, 2).first
+        puts "Symlinking #{f} to #{@basefile_nover}"
+        FileUtils.ln_sf f, @basefile_nover
+        # @basefile_noarch=f.split(/#{gcc_arch}-/, 2).last
+        # puts "Symlinking #{f} to #{@basefile_noarch}"
+        # FileUtils.ln_sf f, @basefile_noarch
+        @basefile_noarch_nover = @basefile_nover.split(/#{gcc_arch}-/, 2).last
+        puts "Symlinking #{f} to #{@basefile_noarch_nover}"
+        FileUtils.ln_sf f, @basefile_noarch_nover
+      end
+      Dir.glob("*-#{gcc_version}").each do |f|
+        @basefile_nover = f.split(/-#{gcc_version}/, 2).first
+        puts "Symlinking #{f} to #{@basefile_nover}"
+        FileUtils.ln_sf f, @basefile_nover
+      end
+      # many packages expect this symlink
+      puts "Symlinking gcc-#{gcc_version} to cc"
+      FileUtils.ln_sf "gcc-#{gcc_version}", 'cc'
+    end
+    # make sure current version of gcc LTO plugin for Gold linker is installed.
+    FileUtils.mkdir_p "#{CREW_LIB_PREFIX}/bfd-plugins/"
+    puts "Symlinking #{CREW_PREFIX}/libexec/#{gcc_dir}/liblto_plugin.so to #{CREW_LIB_PREFIX}/bfd-plugins/"
+    FileUtils.ln_sf "#{CREW_PREFIX}/libexec/#{gcc_dir}/liblto_plugin.so", "#{CREW_LIB_PREFIX}/bfd-plugins/"
+    # binutils makes a symlink here, but just in case it isn't there.
+    unless CREW_LIB_PREFIX.to_s == "#{CREW_PREFIX}/lib"
```

Comment:
Would `if ARCH == 'x86_64'` be easier to understand?

Reply:
For sure... I just wanted a bit about the reasoning, but probably simpler just to have that in there.